Gopkg.toml

```diff
@@ -1,10 +1,10 @@
 [[constraint]]
   name = "github.com/gogo/protobuf"
-  version = "0.5.0"
+  version = "1.0.0"
```

Comment:
We could go as high as 1.2.1. Internal is on 1.0.0.

Reply:
I am hoping to bump internal to 1.2.1. I can send you that pr.